### PR TITLE
Release automation pipeline fix

### PR DESCRIPTION
**Pull request by fredtibbitts**
# Description

Fix the stage 2 release pipeline to login to GitHub before running the script to create the github release.

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

**Review by anders-swanson — APPROVED** [review]

File: release/builds/JenkinsfilePostPRT [hunk 1]
```diff
@@ -131,7 +131,8 @@ pipeline {
             steps {
                 script {
                   sh """
-                    export GITHUB_TOKEN=
+                    echo "${env.GITHUB_ACCESS_TOKEN}" | gh auth login --with-token
+
                     cd ${WORKSPACE}/release/scripts
                     ./create_github_release.sh ${params.RELEASE_VERSION} ${params.RELEASE_COMMIT} ${RELEASE_BINARIES_DIR} ${TEST_RUN}
```
Comment (bhatpmk) [inline release/builds/JenkinsfilePostPRT:137]: Authentication with a GitHub host - gh auth login, is done in create_github_release.sh. May I know the reason for doing this again in Jenkins script ?

Comment (fredtibbitts) [reply release/builds/JenkinsfilePostPRT:137]: These changes are on the release-1.0 branch, and in that branch the create_github_release.sh script does not have the `gh auth login` command. I see that it is in master though. I prefer having the login in the jenkinsfile but I think either way is fine. If you think it belongs in the script then I'll update this to match what is in master.

Comment (bhatpmk) [reply release/builds/JenkinsfilePostPRT:137]: I did not realize that it is not there in the release-1.0 branch. I am fine doing this in Jenkinsfile, probably we can change master accordingly.

**Review by bhatpmk — APPROVED** [review]

File: release/builds/JenkinsfilePostPRT (hunk 1 above)
Comment (fredtibbitts) [reply release/builds/JenkinsfilePostPRT:137]: Yes, I'll push another PR with the changes for master, and I'll make this consistent. Thanks.

**Pull request merged**